chat-widget/src/components/livechatwidget/common/startChat.ts

```diff
@@ -81,9 +81,11 @@ const initStartChat = async (chatSDK: any, chatConfig: ChatConfig | undefined, g
     try {
         const authClientFunction = getAuthClientFunction(chatConfig);
         if (getAuthToken && authClientFunction) {
+            dispatch({ type: LiveChatWidgetActionType.SET_CONVERSATION_STATE, payload: ConversationState.Loading });
             // set auth token to chat sdk before start chat
             const authSuccess = await handleAuthentication(chatSDK, chatConfig, getAuthToken);
             if (!authSuccess) {
+                dispatch({ type: LiveChatWidgetActionType.SET_CONVERSATION_STATE, payload: ConversationState.Closed });
```

Comment:
If there is a prob is auth token, and the state is set to close, I think it might show the Start Chat button. For customer, it would give an experience that Start chat button wasn't clicked, so customer will keep on clicking on it. Can you verify this behavior in embed mode? How about using `inactive` state or adding a new state?

Reply:
yep it is showing chat button. How about we use loading pane instead?

Reply:
can we include some screen shots how it looks in the scenario of failure ?